frontend/src/board/pgn/boardTools/underboard/settings/GameSettings.tsx

```diff
@@ -38,12 +42,36 @@ interface GameSettingsProps {
 
 const GameSettings: React.FC<GameSettingsProps> = ({ game, onSaveGame }) => {
     const isFreeTier = useFreeTier();
-    const [visibility, setVisibility] = useState(game.unlisted ? 'unlisted' : 'public');
+    const [visibility, setVisibility] = useState(
+        game.unlisted ?? true ? 'unlisted' : 'published',
+    );
     const [orientation, setOrientation] = useState<BoardOrientation>(
         game.orientation ?? 'white',
     );
+    const [dirty, setDirty] = useState(false);
```

Comment:
dirty should be calculated from the state, not a state variable itself. What if I change the orientation and then immediately change it back? The current code will still show as dirty, even though it actually isn't

Reply:
It was originally that way and I dislike the shortcoming you mentioned, but the upstream dependencies were not keeping in sync and I was out of time. I will revisit to see if I can find a workaround or better articulate the problem.

Reply:
essentially updating chess headers (form useChess) was not keeping in sync with game (I think from useGame), both firmly outside of the code I was modifying so I declared "here be dragons" and moved on. I'll look back into it in case there was something I was missing.